example-gto/code/mlem/.mlem/config.yaml

```diff
@@ -0,0 +1,6 @@
+#core:
```

Comment:
dead code?

Reply:
this tells mlem to store state on s3 so it is persisted between GHA jobs. We should enable it, it's in TODOs

Reply:
so, let's make it an extra PR when it's ready?

Reply:
I dont see why, we should do it here (I actually already did this in my fork, I guess @aguschin just don't have aws creds)